docs/learn/tokenomics2/tokenomics2-network-fees.md

```diff
@@ -134,4 +134,33 @@ The values for the parameters above are set so that EVM fee and the Native fee a
 
 ## Fee Alignment Transition Period
 
-Legacy Astar Network tokenomics fee model was not aligned between the two systems - same resource consumption via native or Ethereum transactions resulted in significantly different fees. To allow network stakeholders to adjust to the Tokenomics 2.0 fee model, alignment of fees between the two systems will be gradually introduced once the change is enacted (live) on the network.
\ No newline at end of file
+Legacy Astar Network tokenomics fee model was not aligned between the two systems - same resource consumption via native or Ethereum transactions resulted in significantly different fees. To allow network stakeholders to adjust to the Tokenomics 2.0 fee model, alignment of fees between the two systems will be gradually introduced once the change is enacted (live) on the network.
+
+## Hybdrid Inflation model
```

Comment:
@PierreOssun I think it would be better of to create a new sub page named Infation (under Tokenomics 2.0) and than add this content there - now it is at the bottom of the page called Fee Model - and Inflation is a separate topic (all connected under Tokenomics 2.0 umbrella)